azavea/iow-boundary-tool
# Add ability to resubmit boundary

## Overview

This PR adds the ability to resubmit boundaries.

Closes #180 

### Demo

https://user-images.githubusercontent.com/8356789/200381497-6c7a426c-6e95-476a-8a63-db4c69fb7e4f.mov

### Notes

Showing annotations on the next draft required returning previous submission info from the boundary details route. 

## Testing Instructions

- Login as c1@azavea.com
- Select "Azavea Test Utility"
- Go to the submission in status "Needs Revisions"
  - [x] Ensure "go to draw page" button reads "Revise map"
  - [x] Ensure clicking this button starts a new submission and redirects you to the draw page
  - [x] Ensure the shape from the last submission is populated by default
  - [x] Ensure annotations from the last submission's review are visible and read-only
- Make some changes to the shape
- Click "Review and submit" and go through the modals
  - [x] Ensure the boundary is now in "Submitted" status and has the correct shape/comments

## Checklist
